Alexey/Text/text_editor/main/text_editor.py

```diff
@@ -0,0 +1,34 @@
+def main():
+    command = str(input('Enter command: r/w/a or help '))
```

Comment:
Put it in a while loop and do it internal
Add command like '/q' to exit from the loop